guides/source/configuring.md

```diff
@@ -2126,7 +2126,7 @@ Specifies a default value for the `decoding` attribute of `<img>` tags rendered
 
 #### `config.action_view.annotate_rendered_view_with_filenames`
 
-Determines whether to annotate rendered view with template file names. This defaults to `false`.
+Determines whether to annotate rendered view with template file names. This defaults to `true`.
```

Comment:
This should stay as false. It only gets set to true if you use the templated config file (and that’s only in one environment) but otherwise the code default is false.

Reply:
Got it! The docs are about the specific setting, not about what value it takes (in development).
Thank you!